### PR TITLE
Add CN Translation 

### LINKED ISSUE #633: Translating exiting files (any language allowed)
I've decided to add translation of existing files into different languages.

**How to Add Translation:**

1. In the main readme, if the desired language is not available, follow these steps:

   - Add a new section for the language. For example, if you want to add a Hindi translation, follow the steps below.

2. Add a language code to the existing file. For instance, if the file name is "readme.md," change it to "readme_hi.md" for Hindi language.

3. Modify the hyperlinks for the added language accordingly.

**Adding a New Language:**

To add a new language, follow these steps:

1. Go to the main readme where you will find the language section, which may look like this:
 ```html
   <h4 align="center">
       <p>
           <b>English</b> |
           <a href="https://github.com/shhossain/computer_science/blob/main/README_bn.md">বাংলা</a>
       </p>
   </h4>
```
2. To add your language (e.g., Hindi), modify it as follows:
```html
<h4 align="center">
    <p>
        <b>English</b> |
        <a href="https://github.com/shhossain/computer_science/blob/main/README_bn.md">বাংলা</a> |
        <a href="https://github.com/shhossain/computer_science/blob/main/README_hi.md">Hindi</a>
    </p>
</h4>
```

## PRs:
- #630

## Currently Working [ Not updated regularly. Check if it's already added in the main branch ]
- Tamil (@Rathish-Rajendran) #650 [5% complete]
- Español (@Wandererr01) #635 [1% complete]
- Français(@exrol) #637 [2% complete]
- ગુજરાતી(@anubhav1450) #655 [5% complete]

### PR DESCRIPTION
**Pull request by jiangfz1997**
## Proposed changes
Add Readme CN translation for Chinese users. Resolve #633

## Types of changes

What types of changes does your content introduce to this project?
_Put an `x` in the boxes that apply (without "`")_

- [ ] Proofreading (fix spelling, grammar, punctuation, accuracy) [Proofreading](https://www.scribbr.com/language-rules/what-is-proofreading/)
- [x] New content (new section, new readme, etc.)
- [x] Modifications (modifying existing content)
- [ ] Other (please describe)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. Don't hesitate to ask if you're unsure about any of them. We're here to help! This simply reminds us of what we will look for before merging your code._
_Note: If you submit code make sure to test it_

- [ ] Tested any code that was added/modified in this PR
- [ ] Added external links
- [ ] Added internal links
- [ ] Added images
- [ ] Added videos
- [ ] Added code snippets

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Adding your name to the contributor's list
Comment "@all-contributors please add @your_username for content" to add your name to the contributor's list.
_You can change the "content" with any of [this](https://allcontributors.org/docs/en/emoji-key)_

### Reviewers: 
- [ ] @username

_Please ensure that you are not adding any sensitive information like API keys, secrets, passwords, tokens, usernames, or other credentials. Refrain from adding your name, email, or other personal information._

### CODE REVIEW

File: README_cn.md [hunk 1]
```diff
@@ -0,0 +1,939 @@
...
+        <b>简体中文</b>
+    </p>
+</h4>
+
+## 贡献指南
+如果你对于贡献这个项目感兴趣，请查看 [CONTRIBUTING.md](https://github.com/shhossain/computer_science/blob/main/CONTRIBUTING.md) 以了解如何对项目做出贡献。非常感谢你的贡献！
+
+## 目录
+
+- [介绍](#introduction)
```
Comment (shhossain) [inline README_cn.md:24]: You need to fix all of the items' links in the table of contents.
FROM:

\[介绍\]\(\#introduction\)

TO:
\[介绍\](\#介绍\)

Do this for all of the items.

**Review by shhossain — CHANGES_REQUESTED** [review]

**Comment by jiangfz1997** [comment]
Hi @shhossain,
Sorry for the late reply, I just fixed the intro issue now it should be able to redirect to the title for each paragraph.

**Review by shhossain — APPROVED** [review]
@jiangfz1997 No problem. Merging.

**Comment by shhossain** [comment]
@all-contributors add @jiangfz1997 for content.

**Pull request merged**